MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -666,6 +691,15 @@ public static bool IsTextFormat(string format)
                    && !string.Equals(codec, "dvb_subtitle", StringComparison.OrdinalIgnoreCase);
         }
 
+        public static bool IsPgsFormat(string format)
+        {
+            string codec = format ?? string.Empty;
+
+            return codec.Contains("pgs", StringComparison.OrdinalIgnoreCase)
+                   || string.Equals(codec, "sub", StringComparison.OrdinalIgnoreCase)
+                   || string.Equals(codec, "sup", StringComparison.OrdinalIgnoreCase);
```

Comment:
I think the file extension used by PGSSUB is [`.SUP`](https://github.com/FFmpeg/FFmpeg/blob/8d6014dbc6fb7c324a114030761901220ebc3540/libavformat/supdec.c#L104). While the [`.SUB/.IDX`](https://github.com/FFmpeg/FFmpeg/blob/8d6014dbc6fb7c324a114030761901220ebc3540/libavformat/mpeg.c#L1041) pairs is used by VOBSUB/DVDSUB.

ffmpeg can infer the codec from the file content, even if the file extension is wrong. For example:
```
Input #0, sup, from '1.sub':
  Duration: N/A, start: 0.000000, bitrate: N/A
  Stream #0:0: Subtitle: hdmv_pgs_subtitle, 1920x1080
```

```
Input #0, sup, from '1.file':
  Duration: N/A, start: 0.000000, bitrate: N/A
  Stream #0:0: Subtitle: hdmv_pgs_subtitle, 1920x1080
```